packages/contract-artifacts/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import * as AssetProxyOwner from '../artifacts/AssetProxyOwner.json';
+import * as DummyERC20Token from '../artifacts/DummyERC20Token.json';
+import * as DummyERC721Token from '../artifacts/DummyERC721Token.json';
+import * as ERC20Proxy from '../artifacts/ERC20Proxy.json';
+import * as ERC20Token from '../artifacts/ERC20Token.json';
+import * as ERC721Proxy from '../artifacts/ERC721Proxy.json';
+import * as ERC721Token from '../artifacts/ERC721Token.json';
+import * as Exchange from '../artifacts/Exchange.json';
+import * as Forwarder from '../artifacts/Forwarder.json';
+import * as IValidator from '../artifacts/IValidator.json';
+import * as IWallet from '../artifacts/IWallet.json';
+import * as OrderValidator from '../artifacts/OrderValidator.json';
+import * as WETH9 from '../artifacts/WETH9.json';
+import * as ZRXToken from '../artifacts/ZRXToken.json';
+
+export {
+    AssetProxyOwner,
```

Comment:
Note that we don't have to do the `(as any) as ContractArtifact` hack anymore! With `"resolveJsonModule": true` in __tsconfig.json__ TypeScript supports typed JSON imports. It also works with Project References, so when you watch, it will recompile when the JSON file(s) change.